From d9870a8096978172710466b937bf05b7f7f7a9f9 Mon Sep 17 00:00:00 2001
From: "Owen W. Taylor" <otaylor@fishsoup.net>
Date: Tue, 22 Apr 2014 21:09:58 -0400
Subject: [PATCH 2/2] Add an autogen.sh that calls configure

Even though the the only command that needs to be run is autoconf,
having a standard name for the script that generates configure is
useful for automated build systems. See:

https://people.gnome.org/~walters/docs/build-api.txt
---
 autogen.sh | 3 +++
 1 file changed, 3 insertions(+)
 create mode 100755 autogen.sh

diff --git a/autogen.sh b/autogen.sh
new file mode 100755
index 0000000..bf42772
--- /dev/null
+++ b/autogen.sh
@@ -0,0 +1,3 @@
+#!/bin/sh
+
+autoconf
-- 
1.9.0

